dist/index.html

```diff
@@ -26,10 +26,10 @@
       </nav>
     </header>
     <main>
-      <section id="home-view">
-        <figure id="home-banner">
-          <img id="recipe-of-the-day" alt="Recipe of the Day" src="https://spoonacular.com/recipeImages/595736-556x370.jpg">
-          <figcaption id="recipe-today"><h1>Recipe of the Day</h1></figcaption>
+      <section class="home-view">
+        <figure class="home-banner">
+          <img alt="Recipe of the Day" src="https://spoonacular.com/recipeImages/595736-556x370.jpg">
+          <figcaption><h1>Recipe of the Day</h1></figcaption>
```

Comment:
We need to remove this code. When refreshing we see the picture of a cookie for a split second... Since we're now populating this with JS we no longer need this.

Reply:
Ooh great catch!